index.bs

```diff
@@ -4415,15 +4421,14 @@ An [=interface=] can be declared to be
 </pre>
 
 Objects implementing an interface that is declared to be maplike
-represent an ordered list of key–value pairs known as its <dfn id="dfn-map-entries" export>map entries</dfn>.
+represent an [=ordered map=] of key–value pairs, initially empty,
+known as its <dfn id="dfn-map-entries" export>map entries</dfn>.
```

Comment:
```suggestion
known as that object's <dfn id="dfn-map-entries" export>map entries</dfn>.
```

here and below. (To make it clear _each platform object implementing a maplike interface_ owns a map entries, not the maplike interface itself.)